tests/unit/components/setup.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * The tests for our components need React, because we are using the new
+ * technology of shallow rendering, so we are passing to the renderer
```

Comment:
Both `TestUtils.renderIntoDocument` and `loadChild` would've required the use of JSX.